docs/src/pages/guides/migration-v4/migration-v4.md

```diff
@@ -45,6 +45,12 @@ yarn add @material-ui/core@next
 
 ## Handling breaking changes
 
+### non-ref-forwarding class components
+
+Support for these components in the `component` prop or as an immediate `children` has been dropped. If you were using `unstable_createStrictModeTheme` or didn't see any warnings related to `findDOMNode` in `React.StrictMode` then you don't need to do anything.
```

Comment:
It took some time to understand what these reverted to.

```suggestion
Support for these components (non-ref-forwarding class components) in the `component` prop or as an immediate `children` has been dropped. If you were using `unstable_createStrictModeTheme` or didn't see any warnings related to `findDOMNode` in `React.StrictMode` then you don't need to do anything.
```

Reply:
Removing the reference to `this` altogether